moviesTime/Network/NetworkProvider.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  NetworkProvider.swift
+//  moviesTime
+//
+//  Created by Jose Eduardo on 30/07/21.
+//
+
+import Foundation
+import Alamofire
+
+final class NetworkProvider{
+    
+    static let shared = NetworkProvider()
+    
+    private let kBaseURL = "https://api.themoviedb.org/3/movie/"
```

Comment:
this sensitive data shouldn't be here, you can use a `.plist` file to store all these values. The best thing is to encrypt that file and then decrypt it when you will use it, for now you can use just the file without encryption